spec/mdn/mdn.mdown

```diff
@@ -0,0 +1,88 @@
+# Introduction
+
+JMAP <https://tools.ietf.org/html/draft-ietf-jmap-core-06> is a generic protocol for synchronising data, such as mail, calendars or contacts, between a client and a server. It is optimised for mobile and web environments, and aims to provide a consistent interface to different data types.
+
+This specification defines a method helping to send MDN (Message Disposition Nodification) messages. MDN are defined in [@!RFC8098] and are used as "read receipts", "acknowledgements", or "receipt notifications".
+
+## Notational conventions
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [@!RFC2119].
+
+Type signatures, examples and property descriptions in this document follow the conventions established in Section 1.1 of <https://tools.ietf.org/html/draft-ietf-jmap-core-06>.
+Email object and methods are defined in <https://tools.ietf.org/html/draft-ietf-jmap-mail-06>.
+
+## Terminology
+
+The same terminology is used in this document as in the core JMAP specification.
```

Comment:
link to the JMAP core spec reference draft?

Reply:
it's already linked in several places (and here also it's a copy paste from other specs)